beta-src/src/components/map/components/WDTerritory.tsx

```diff
@@ -157,14 +247,16 @@ const WDTerritory: React.FC<WDTerritoryProps> = function ({
           );
         })}
       {territoryMapData.unitSlots &&
-        territoryMapData.unitSlots.map((unitSlot) => (
+        territoryMapData.unitSlots.map(({ name, x, y }) => (
           <WDUnitSlot
-            key={unitSlot.name}
-            name={unitSlot.name}
+            key={name}
+            name={name}
             territoryName={territoryMapData.name}
```

Comment:
name and territoryMapData.name not the same?

Reply:
One is the name of the unit slot, the other is the name of the territory.